.github/workflows/test.yml

```diff
@@ -43,3 +43,9 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           format: golang
           file: tests/output/cover.out
+
+      - name: Test Summary
+        uses: test-summary/action@v2
+        with:
+          paths: "tests/output/*-tests.xml"
+        if: always()
```

Comment:
why is this if needed over here?

Reply:
it's something [they recommended in their docs](https://github.com/test-summary/action?tab=readme-ov-file#getting-started):
> Note the `if: always()` conditional in this workflow step: you should always use this so that the test summary creation step runs even if the previous steps have failed. This allows your test step to fail -- due to failing tests -- but still produce a test summary.